src/interfaces/component.ts

```diff
@@ -1,5 +1,3 @@
-import { FastDomNode } from './node';
-
 export interface ClassConstructor<T> {
   new (inputs?: ComponentsInputs): T;
```

Comment:
Here must be `...args: Array<any>` because right now we accept multiple args.

Reply:
And for better type checking we can write `ClassConstructor<T extends Component>`
and in `generators/components.ts`
```
const instance = new classProvider(...args) // remove as any
```

Reply:
and 
`export function createComponent<T extends Component>(`

Reply:
Definitions updated here: d9ba1bd